docs/reference/errors-and-warnings/NU1506.md

```diff
@@ -0,0 +1,43 @@
+---
+title: NuGet Error NU1506
+description: NU1506 error code
+author: nkolev92
+ms.author: nikolev
+ms.date: 03/31/2022
+ms.topic: reference
+f1_keywords: 
+  - "NU1506"
+---
+
+# NuGet Error NU1506
+
+## Scenario 1
+
+<pre>Duplicate 'PackageVersion' items found. Remove the duplicate items or use the Update functionality to ensure a consistent restore behavior. The duplicate 'PackageVersion' items are: X [1.0.0], X [2.0.0].</pre>
+
+### Issue
+
+Sometimes when adding `PackageVersion` items, you may end up adding an item for a particular package multiple times.

```

Comment:
Is `<PackageVersion />` deprecated ? https://github.com/Microsoft/MSBuildSdks/tree/main/src/CentralPackageVersions#version-20-breaking-changes 

If so, should we write that in this documentation page?

Reply:
You are conflating NuGet's native CPM implementation and the MSBuild SDK one. 
PackageVersion is very much not deprecated in NuGet's CPM.